news/2017-08-17-happystick-summer-tour-concludes.md

```diff
@@ -0,0 +1,50 @@
+
+[Full match information](https://osu.ppy.sh/community/matches/35252862)
+
+![HappyStick Summer Tour Final Results](http://i.imgur.com/3mqZcvM.png)
+
+For full results including all ranking tiers, check out the tournament's brackets [here](http://challonge.com/users/happystick). 
+
+Congratulations to all 13 victors in the HappyStick Summer Tour who will receive their share of the prize pool and a unique profile badge in addition to being able to lay claim to the title of HappyStick Tour Champion until the next Tour event!
+
+Congratulations as well to all the Grand Finalists who will also receive prize money to recognize their impressive performance, and to all the third and fourth place finishers who will receive prize money and osu! Supporter rewards, respectively.
+
+If you're interested in more information about the tournament, you can find it [here](https://docs.google.com/document/d/1f2FUWtksvAd95y56CIu7cCyleg0nqeWeva7WB_Y1Wq4/pub); Finals Mappools are available [here](https://docs.google.com/spreadsheets/d/1DDv3RRV7xkxC9p6fXABtk7dlZZHWGf_4hPpOr7XHSSA/pubhtml), and on [HappyStickTour.com](http://happysticktour.com/public/) you can find public data such as lobby information and mappool statistics. 
+
+
+
```

Comment:
tradition dictates you get to "sign" your name with a dash here, you can do so if you like.